### DISCUSSION TITLE
TABKE NOT FOUND<title>

### DISCUSSION THREAD

**Discussion by LEMBEN-1** [community/community#121557] [Programming Help]
### Select Topic Area

Product Feedback

### Body

I get an error message saying the table does not exist even after selecting the correct Data Base

**Comment by ebndev** [comment]
Hi @LEMBEN-1, we're happy you're here! 

You are more likely to get a useful response if you are explicit about what your project entails, like what database you are using and the error log. Giving a few more details might help someone give you a nudge in the right direction.
